services/QuillLMS/app/models/subscription.rb

```diff
@@ -344,7 +346,7 @@ def self.set_cb_lifetime_expiration_and_start_date
   end
 
   def self.create_with_school_or_user_join school_or_user_id, type, attributes
-    type.capitalize!
+    type = type.capitalize
```

Comment:
What's the purpose of this change?

Reply:
@brendanshean I haven't requested reviews yet as I'm going to push up a couple more commits. But this change is because the `type` argument is not a frozen string, so it cannot be modified.